build.sbt

```diff
@@ -21,7 +21,8 @@ lazy val alchemy = (project in file("."))
     libraryDependencies ++= Seq(
       scalaTest % Test withSources(),
       "net.agkn" % "hll" % "1.6.0" withSources(),
-      "org.postgresql" % "postgresql" % "42.2.8" % Test withSources()
+      "org.postgresql" % "postgresql" % "42.2.8" % Test withSources(),
+      "org.apache.spark" %% "spark-sql" % sparkVersion % "provided" withSources()
```

Comment:
Why isn't this in `sparkDependencies`?

Reply:
@ssimeonov - good catch.  Pushed up another commit to refactor this even more.  We don't need the test build, so we can just put all the project dependencies in one place.  Let me know if this looks better!